src/index.html

```diff
@@ -0,0 +1,12 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <link rel="stylesheet" href="css/style.css" />
+    <script src="scripts/index.js" async defer></script>
```

Comment:
type='module' give us possibility to use ES modules feature, so add it, please. Also js should be after title tag.

Reply:
I forgot it, done.